tox.ini

```diff
@@ -6,15 +6,15 @@
 [tox]
 envlist =
     py36
+    py37
     docs
     style
 
 [testenv]
 description = Run unit tests with pytest
 extras = voice, test, mongo
 deps =
-    https://github.com/Rapptz/discord.py/archive/7eb918b19e3e60b56eb9039eb267f8f3477c5e17.zip#egg=discord.py[voice]
-    -rrequirements.txt
+    https://github.com/Rapptz/discord.py/archive/f06563cb3520058c900ba6d4921ce38f2407b729.zip#egg=discord.py
```

Comment:
This wont work as a dependency link anymore. Something changed on github's redirection and this breaks now.

```
fatal: unable to update url base from redirection:
  asked for: https://github.com/Rapptz/discord.py/tarball/f06563cb3520058c900ba6d4921ce38f2407b729/info/refs?service=git-upload-pack
   redirect: https://codeload.github.com/Rapptz/discord.py/legacy.tar.gz/f06563cb3520058c900ba6d4921ce38f2407b729
```
Link format should no longer point to an archive, but a commit as part of a repo.
```
git+https://github.com/Rapptz/discord.py@f06563cb3520058c900ba6d4921ce38f2407b729#egg=discord.py1.0.0a

Or we need to adjust the archive link and hope it doesn't change again. I don't like relying on that, and the redirection behavior to remain the same.

Reply:
I don't seem to be able to reproduce the issue.

I'm not really sure if `git+` links work as dependency links for a package in setup.py. But even if they did, they require all users to have git installed when installing the bot.

Reply:
I went to get full reproduction steps and can't anymore either. I'll keep an eye out for if it comes up again in some way later on at which point it can be addressed, but until I can reproduce it even semi-consistently, this can be ignored for now. 

Worth noting, git is a listed dependency in all our documentation and installation instructions due to downloader's use of it.